mkdocs.yml

```diff
@@ -21,6 +21,7 @@ repo_url: https://github.com/aristanetworks/ansible-avd
 use_directory_urls: true
 nav:
   - Home: README.md
+  - Release Notes: docs/release-notes.md
```

Comment:
Should we create a Release Note section per major versions like `1.0.x` and `1.1.x` as it might become difficult to read after few versions?

Reply:
@titom73 that is a good suggestion that makes sense to me. I will update it.
Also, any suggestions on the MKDOCs link check CI, will that impact the actual release to github.io?